src/Controller/Api/LinkedCentersController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller\Api;
+
+use App\Api\Manager\ApiClientManager;
+use App\ControllerV2\AbstractController;
+use App\Entity\Centre;
+use App\Repository\CentreRepository;
+use Doctrine\ORM\NonUniqueResultException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Attribute\AsController;
+
+#[AsController]
+final class LinkedCentersController extends AbstractController
+{
+    public function __invoke(
+        Request $request,
+        ApiClientManager $apiClientManager,
+        CentreRepository $repository
+    ): ?Centre {
+        try {
+            return $repository->findByDistantId(
+                $request->attributes->getInt('id'),
+                $apiClientManager->getCurrentOldClient()?->getRandomId()
+            );
+        } catch (NonUniqueResultException) {
+            return null;
+        }
```

Comment:
This route will be used to check if an organization is linked, so I don't think we should return null if more than one result.